client/landing/gutenboarding/components/header/index.tsx

```diff
@@ -139,6 +142,13 @@ const Header: React.FunctionComponent = () => {
 	const hasPlaceholder =
 		!! siteTitle && ! recommendedDomainSuggestion && previousRecommendedDomain !== '';
 
+	const onDomainSelect = ( suggestion: DomainSuggestion | undefined ) => {
+		trackEventWithFlow( 'calypso_newsite_select_domain', {
+			domain_name: suggestion?.domain_name,
```

Comment:
Can text value like this used in Tracks meaningfully? Seems like they'd be unique each time?

Ok to include now and clean out later if we find out they're not useful. :-)

Reply:
We [do it in the domains step](https://github.com/Automattic/wp-calypso/blob/master/client/signup/steps/domains/index.jsx#L704) for `/start`, but yeah, I can't imagine how it would be useful.

`tld: 'co.uk'` or `isFreeDomain: false` I can see being generally more interesting.

Reply:
Yes, TLDs, `isFreeDomain`, `isRecommended` etc. I think we tracked something like that with "Traintracks"? I'm not very familiar with that tho.